### ISSUE TITLE
Accessibility issue in Relational Databases Certification

### ISSUE THREAD

**Issue by arqamgrt** [freeCodeCamp/freeCodeCamp#47258]
Hello Campers. Hope you’re doing well.
I’m Facing an issue in Relational Databases Certification (online).
There is no output spoken by screen readers in the terminal of vs code. I know, I can use it locally. I actually did it. But in the PostGreSQL Course, I was stuck on first step. I don’t know why, but it was failing the test. Same steps I tried online, and it was successful. But again, in online, there is no speech output for screen reader users of any command you type in the terminal.
I’m looking for a possibility to solve this accessibility issue. if not, then at least, in local, the bug should be fix. So, I’ll be able to complete the certification.
Looking forward to see your valuable responses on this issue.

**Comment by bbsmooth** [comment]
@arqamgrt I have not looked at this course yet but I will be happy to give it a try. Can you give us some more details about exactly where you are having problems. Since I have no experience with this course, please be as specific as possible. Also, what operating system, browser, and screen reader are you using?

**Comment by ShaunSHamilton** [comment]
Hello there,

Thank you for taking the time to open this. Unfortunately, we have no control over the VM serving the VSCode-style editor. I will ping @moT01 as we might need to transfer this issue to CodeAlly?

**Comment by arqamgrt** [comment]
> @arqamgrt I have not looked at this course yet but I will be happy to give it a try. Can you give us some more details about exactly where you are having problems. Since I have no experience with this course, please be as specific as possible. Also, what operating system, browser, and screen reader are you using?

Well, I'm using Windows 10 21H2. with chrome browser and NVDA screen reader.
if you talk about online mode, the only problem is that I can't hear the output of anything on terminal.

**Comment by moT01** [comment]
Thanks for reporting @arqamgrt, sorry for the delay here. I was sort of able to reproduce this with voiceover on mac. It reads output from the actual terminal, but not the terminal in VS Code - both locally and in CodeAlly. So when you use the terminal in your local VS Code, it reads the output for you? If so, do you know why I am not getting that? Maybe some extension or program you have, or maybe it's NVDA.

I will see if I can reproduce the issue you had with running these locally and get back to you.

**Comment by arqamgrt** [comment]
@moT01 Are you trying on Windows? or mac. If you want to try it on windows, then use NVDA for this.
but in local, I have a problem. in the PostGreSQL course, first step is not working in local. same step if working fine in code ally.

**Comment by moT01** [comment]
Right, I will look into that @arqamgrt. I am on a mac.

So, to clarify, do you get the terminal output read when using the terminal in VS Code anywhere? I'm trying to figure out if it's something with VS Code or CodeAlly.

**Comment by arqamgrt** [comment]
no no, I know this. vs code is completely fine here. the problem is only with codeAlly. in vs code, everything works fine including terminal.

**Comment by moT01** [comment]
I loaded the tutorial up locally for the first time in a while. I was unable to reproduce the getting stuck part of this issue. Sometimes it happens. There's some [troubleshooting steps to try here](https://forum.freecodecamp.org/t/running-the-relational-database-curriculum-in-your-browser/500231#issues-within-a-course-9) - otherwise, maybe just start over. The forum is the place to troubleshoot that, so if you are still stuck - I would make a post over there.

I am going to transfer this issue to the repo we have for tracking RDB issues that aren't fixable by fCC.
